### DISCUSSION TITLE
Need help inserting image.

### DISCUSSION THREAD

**Discussion by KendallDoesCoding** [TechStudent10/techstudent10#8] [General]
Hey @TechStudent11 ,

I wanted to add this image on my website near my name, I've tried various codes, but the picture side by side to the text but a little below, I want the picture on the same line.

My website is at: https://kendalldoescoding.gq
Picture URL: https://d2gg9evh47fn9z.cloudfront.net/800px_COLOURBOX33047209.jpg
Github Repo: https://github.com/KendallDoesCoding/portfolio

If you can help me on this, it would be really helpful. Please adjust the code on the github if you find a solution to my problem and I will update it on my website.

Thanks a lot,
Kendall Does Coding

**Comment by KendallDoesCoding** [comment]
When I tried, it was overlapping with the Youtube and Github icon.

**Reply by TechStudent10** [reply]
Hm okay I'll try.

**Reply by KendallDoesCoding** [reply]
thanks

**Comment by KendallDoesCoding** [comment]
@TechStudent11 Reminder on this.

**Comment by KendallDoesCoding** [comment]
So, do you know how to do this @TechStudent11

**Reply by TechStudent10** [reply]
Saved ✔️
